src/prActions.ts

```diff
@@ -0,0 +1,55 @@
+import { addComment, needsUpdate, removeLabel, updatePr } from "./github.ts";
+import { debounce } from "https://deno.land/std@0.189.0/async/mod.ts";
+
+const execute = async (
+  label: string,
+  pr: { number: number; user: { login: string } },
+) => {
+  switch (label) {
+    case "giteabot/update-branch": {
+      const err = await updateBranch(pr);
+      if (err?.message == "failed to sync PR") {
+        // notify maintainer about failed update
+        await addComment(
+          pr.number,
+          `Failed to update branch :tea:`,
+        );
+      }
```

Comment:
Logging the message is enough, don't comment

Reply:
Okay, removed